skpro/regression/cyclic_boosting.py

```diff
@@ -0,0 +1,315 @@
+"""Cyclic boosting regressors.
+This is a interface for Cyclic boosting, it contains efficient,
+off-the-shelf, general-purpose supervised machine learning methods
+for both regression and classification tasks.
+Please read the official document for its detail
+https://cyclic-boosting.readthedocs.io/en/latest/
+"""
+
+"""License.
+EPL 2.0
```

Comment:
The interfaces in sktime/skpro should be released under the BSD-3 license of the repo.
Of course the `cyclic_boosting` package can be subject to a different license.

I would think this should be ok, as there is no substantial logic, only interface logic, in the `skpro` module? Comments appreciated, especially if you think otherwise.

Btw, the docstring of the class is what this appears as in the doc, so I would recommend to give more credit to the upstream package, see for instance the `mapie` regressor.

Reply:
I agree. As I also commented before, we should not copy code from Cyclic Boosting here, but just call the respective functionality.